Simplified/Book/Models/NYPLBook.h

```diff
@@ -102,4 +102,9 @@
 /// @return The default NYPLBookContentType
 - (NYPLBookContentType)defaultBookContentType;
 
+/// Add a custom expiration date to book if
+/// 1. book is distributed by Axis360
+/// 2. book does not contain an expiration date
+- (void)addCustomExpirateDate:(nonnull NSDate *)date;
```

Comment:
could you rename this to `addBannedBookExpiration` since the expiration is only applicable for a banned book?